src/EditorFeatures/CSharpTest/MoveStaticMembers/CSharpMoveStaticMembersTests.cs

```diff
@@ -2325,6 +2325,38 @@ internal static class Class1Helpers
             await TestMovementNewFileAsync(initialMarkup, expectedResult1, expectedResult2, newFileName, selectedMembers, selectedDestinationName).ConfigureAwait(false);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMoveStaticMembers)]
+        public async Task TestSelectInKeyWordOfMulitFieldDeclaration()
```

Comment:
```suggestion
        public async Task TestSelectInKeywordOfMulitFieldDeclaration()
```

Not a big deal, but we treat "keyword" as one world in Roslyn.